bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/storage/ldb/KeyValueStorageRocksDB.java

```diff
@@ -198,7 +198,7 @@ private RocksDB initializeRocksDBWithBookieConf(String basePath, String subPath,
             int blockSize = conf.getInt(ROCKSDB_BLOCK_SIZE, 64 * 1024);
             int bloomFilterBitsPerKey = conf.getInt(ROCKSDB_BLOOM_FILTERS_BITS_PER_KEY, 10);
             boolean lz4CompressionEnabled = conf.getBoolean(ROCKSDB_LZ4_COMPRESSION_ENABLED, true);
-            int formatVersion = conf.getInt(ROCKSDB_FORMAT_VERSION, 2);
+            int formatVersion = conf.getInt(ROCKSDB_FORMAT_VERSION, 5);
```

Comment:
Can we log a line at info level about this configuration  ? It will be easier to troubleshot upgrades

Reply:
This code doesn't get used at all if `conf/entry_location_rocksdb.conf` or `conf/ledger_metadata_rocksdb.conf` files exist. That was changed in Bookkeeper 4.15 in https://github.com/apache/bookkeeper/pull/3056 (Pulsar 2.11, https://github.com/apache/pulsar/pull/15142).

@eolivelli Is your intention to print out the effective database version? I think it might be challenging to get it right since I believe that the given format version will only be used for new databases. For existing databases, the format is whatever it was at creation time. Just wondering if the log message would cause confusion.

Reply:
Let's handle the possible logging separately. Bookkeeper 4.14 uses RocksDB 6.29 which supports format_version=5 (introduced in [RocksDB 6.6.0](https://github.com/facebook/rocksdb/releases/tag/v6.6.3)). 
Before RocksDB 6.29, version 6.16 was used. format_version=5 support has been around for a long time.